addon-test-support/logger.ts

```diff
@@ -0,0 +1,186 @@
+import axeCore, { AxeResults, NodeResult, RelatedNode, Result } from 'axe-core';
+
+/**
+ * This file is heavily borrowed from https://github.com/dequelabs/react-axe
```

Comment:
Can you make this a permalink to the actual source (so we can go to that location to track changes over time)?

Reply:
Sure.